tests/test_factory_name.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import factory
+
+from pytest_factoryboy import register
+
+
+@register
+class JSONPayloadFactory(factory.Factory):
+    class Meta:
+        model = dict
+
+    name = "John Doe"
+
+
+def test_fixture_name_as_expected(json_payload):
+    """Test that the json_payload fixture is registered from the JSONPayloadFactory."""
+    assert json_payload["name"] == "John Doe"
+
+
+def test_fixture_name_cant_be_determined(testdir):
+    """Test that an error is raised if the fixture name can't be determined."""
+    testdir.makepyfile(
+        """
+        import factory
+        from pytest_factoryboy import register
+
+        @register
+        class JSONPayloadF(factory.Factory):
+            class Meta:
+                model = dict
+
+            name = "John Doe"
+
+        """
+    )
+    res = testdir.runpytest()
+    res.assert_outcomes(errors=1)
```

Comment:
For `pytest<6.0.0` keyword argument for errors is `error` (singular), so we have to write some adapter for the `pytester` or drop support for `pytest<6.0.0` ;)

Reply:
yeah annoying